checkov/main.py

```diff
@@ -35,22 +36,35 @@ def run(banner = checkov_banner):
                         help='Report output format')
     parser.add_argument('-s', '--soft-fail',
                         help='Runs checks but suppresses error code', action='store_true')
+    parser.add_argument('--bc-api-key', help='Bridgecrew API key')
+    parser.add_argument('--repo-id',
+                        help='Identity string of the repository, with form <repo_owner>/<repo_name>')
```

Comment:
@tronxd Maybe add a branch parameter, which has the default value "master"?

Reply:
@schosterbarak WDYT?

Reply:
I agree. Branch should be a parameter. The SaaS should show results only for the default branch (usually master). 
maybe that's a value you can populate from circleci.

Reply:
@schosterbarak adding the branch as an arg to checkov, defaults to master.